libopenshot fails to build since ruby 2.4.1

Hi,

I tried to build latest release 0.1.8 but it fails to build with new ruby 2.4.1 (same with previous 0.1.7 release):

> [ 75%] Swig source
cd /home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/build/src/bindings/ruby && /usr/bin/cmake -E make_directory /home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/build/src/bindings/ruby
cd /home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/build/src/bindings/ruby && /usr/bin/swig -ruby -outdir /home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/build/src/bindings/ruby -c++ -I/usr/include/ImageMagick-6 -I/usr/include -I/usr/include/libopenshot-audio -I/usr/lib/qt5/include -I/usr/lib/qt5/include/QtWidgets -I/usr/lib/qt5/include/QtGui -I/usr/lib/qt5/include/QtCore -I/usr/lib/qt5/./mkspecs/linux-g++ -I/usr/lib/qt5/include/QtMultimedia -I/usr/lib/qt5/include/QtNetwork -I/usr/lib/qt5/include/QtMultimediaWidgets -I/home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/src/../thirdparty/jsoncpp/include -I/home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/src/bindings/ruby -o /home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/build/src/bindings/ruby/openshotRUBY_wrap.cxx /home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/src/bindings/ruby/openshot.i
/home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/src/bindings/ruby/openshot.i:43: Error: Syntax error in input(1).
make[2]: *** [src/bindings/ruby/CMakeFiles/rbopenshot.dir/build.make:67: src/bindings/ruby/openshotRUBY_wrap.cxx] Error 1
make[2]: Leaving directory '/home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/build'
make[1]: *** [CMakeFiles/Makefile2:363: src/bindings/ruby/CMakeFiles/rbopenshot.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: Entering directory '/home/iurt/rpmbuild/BUILD/libopenshot-0.1.8/build'

--- a/src/bindings/ruby/openshot.i
+++ b/src/bindings/ruby/openshot.i
@@ -40,12 +40,11 @@
 %include <std_except.i>
 
 namespace std {
-{
   template<class T> class shared_ptr {
   public:
     T *operator->();
   };
-}}
+}
 
 /* Mark these classes as shared_ptr classes */
 #ifdef USE_IMAGEMAGICK

